.travis.yml

```diff
@@ -76,6 +76,7 @@ before_script:
 script:
   - ./scripts/copy_opencv.sh
   - ./scripts/main_script.sh
+  - ./gradlew build jacocoTestReport
```

Comment:
Do we require this?

Reply:
I think this Plugin is very useful for this Project

Reply:
@naveentherocker How it can be useful?

Reply:
@harshit98 It Creates a code coverage report for tests of a single project in multiple formats.

Reply:
@naveentherocker pls use this instead: `./gradlew connectedAndroidTest jacocoTestReport` and note that this should be done in `https://github.com/fossasia/phimpme-android/blob/development/scripts/main_script.sh`. I can see the stack trace is generated there as well. if you want you can add `jacocoTestReport`.

Reply:
The giff still shows the addition here. @naveentherocker

Reply:
@naveentherocker pls remove this change